components/show_results.py

```diff
@@ -10,6 +10,8 @@
     ElementsIDs,
 )
 
+def feet_per_second_to_meters_per_second(ftps: float) -> float:
```

Comment:
this function is already defined in the code, you should use that and not define it again